app/lib/features/events/pages/event_details_page.dart

```diff
@@ -387,11 +411,51 @@ class _EventDetailPageConsumerState extends ConsumerState<EventDetailPage> {
             title: Text(inDays),
             subtitle: Text('$startDate - $endDate'),
           ),
+          ListTile(
+            leading: const Icon(Atlas.accounts_group_people),
+            title: participantsListUI(ev.roomIdStr()),
+          ),
         ],
       ),
     );
   }
 
+  Widget participantsListUI(String roomId) {
+    return ValueListenableBuilder(
+      valueListenable: eventParticipantsList,
+      builder: (context, eventParticipantsList, child) {
+        if (eventParticipantsList.isEmpty) {
+          return const Text('No participants going');
+        }
+
+        List<Widget> avtarList = [];
+        for (final participantId in eventParticipantsList) {
+          final memberInfo = ref.watch(
+            roomMemberProvider((roomId: roomId, userId: participantId)),
+          );
+          memberInfo.whenData((profileData) {
+            avtarList.add(
```

Comment:
this isn't really a safe way to generate that list, I'd say. I am not sure what exactly happens with the `avatarList` if the `memberInfo` is delayed or comes in later... it probably means it isn't the same order at least ... but what if any of them fails, for e.g. for timeout reasons? It would show the wrong count, wouldn't it?

I think this should instead be something like:

```
List<Widget> avatarList = eventParticipantsList.map((participantId) =>  ref.watch(
            roomMemberProvider((roomId: roomId, userId: participantId)),
          ).when(data: (profileData) => Padding( ... ), error: (e,s) = FallbackAvatar, loading: () => AvatarSkeleton))).toList();
```

Reply:
Ah, good notice. Let me do the needful changes here.